src/negotiation/taker_negotiator.spec.ts

```diff
@@ -0,0 +1,74 @@
+import { Cnd } from "../cnd";
+import { ComitClient } from "../comit_client";
+import { TakerCriteria } from "./order";
+import { TakerNegotiator } from "./taker_negotiator";
+// @ts-ignore
+import {
+  // @ts-ignore
+  mockGetOrderByTradingPair,
+  // @ts-ignore
+  mockGetExecutionParams
+} from "./maker_client";
+
+jest.mock("../comit_client");
+jest.mock("./maker_client");
+
+const defaultCnd = new Cnd("");
+const defaultComitClient = new ComitClient(defaultCnd);
```

Comment:
If this is the reason for making `bitcoinWallet` and `ethereumWallet` optional, then just create two empty mocks here for those two.